File: src/scene/encounter/EncounterNotification.java
package scene.encounter;

import model.Notification;
import scene.SceneID;

public class EncounterNotification {

	private Notification notification;
	private SceneID sceneID;
	
	public EncounterNotification(Notification notification, SceneID sceneID) {
		this.notification = notification;
		this.sceneID = sceneID;
	}
	
	public Notification getNotification() {
		return notification;
	}
	
	public SceneID getSceneID() {
		return sceneID;
	}
	
}

File: web/src/scene/encounter/EncounterNotification.ts
import { Notification } from "../../model/Notification";

export class EncounterNotification {
  private notification: Notification;
  private sceneID: string | null;

  constructor(notification: Notification, sceneID: string | null) {
    this.notification = notification;
    this.sceneID = sceneID;
  }

  getNotification(): Notification {
    return this.notification;
  }

  getSceneID(): string | null {
    return this.sceneID;
  }
}
